api/jobs.py

```diff
@@ -37,11 +38,33 @@
 def valid_transition(from_state, to_state):
     return (from_state + ' --> ' + to_state) in JOB_TRANSITIONS or from_state == to_state
 
-ALGORITHMS = [
-    'dcm2nii',
-    'qa'
+# Represet the general categories of gear function
+@unique
+class Category(Enum):
+    classifier  = 'classifier' # discover metadata
+    converter   = 'converter'  # translate between formats
+    qa          = 'qa'         # quality assurance
+    analytical  = 'analytical' # general purpose
+
+    # Enum strings are prefixed by their class: "Category.classifier".
+    # This overrides that behaviour and removes the prefix.
+    def __str__(self):
```

Comment:
Can we remove this method and use Category.classifier.value, Category.converter.value, etc in lines 57-59?
It seems cleaner to me. 
You avoid the string conversion  in line 142 also.